pykern/pkcli/github_orgmode.py

```diff
@@ -116,17 +117,25 @@ def _open_issues(self, repo):
 class _OrgModeGen(_Base):
     _TITLE = re.compile(r"^(\d{4})-?(\d\d)-?(\d\d)\s*(.*)")
     # strict for now
-    _HTML_URL = re.compile(r"https://github.com/([\w-]+/[\w-]+)/issues/\d+$")
+    _HTML_URL = re.compile(r"https://github.com/([\.\w-]+/[\.\w-]+)/issues/\d+$")
     _NO_DEADLINES_MARK = "ISSUES DO NOT HAVE DEADLINES AFTER THIS " + _Base._COMMENT_TAG
     _CFG = "#+STARTUP: showeverything\n#+COLUMNS: %13DEADLINE %50ITEM %number(Num) %15assignees %TAGS\n"
 
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
         self._org_d = pykern.pkio.py_path(kwargs["org_d"])
-        if "repo" in kwargs:
-            r = self._github.repo_arg(kwargs["repo"])
-            b = f"{r.organization['login']}-{r.name}"
-            self._issues = self._open_issues(r)
+        if "repos" in kwargs:
+            if not kwargs["repos"]:
+                self._error("no repos supplied")
+            if len(kwargs["repos"]) == 1:
+                r = self._github.repo_arg(kwargs["repos"][0])
```

Comment:
var `r` necessary? may be confused with `r` in loop on  line 137

Reply:
It's used three times below.